src/platform/lumin-runtime/platform-factory.js

```diff
@@ -470,26 +478,33 @@ export class PlatformFactory extends NativeFactory {
     }
   }
 
+  _validateAppType (type) {
+    if (this._appConstructors[type] === undefined) {
+      throw new TypeError(`Invalid argument: Unknown app type: ${type}`);
+    }
+  }
+
+  _validateAppTimeDelta (delta) {
```

Comment:
I'd prefer if both of these were optional parameters (and just defaulted to type=Landscape, delta=0) but that's just my own preference (and a change that could be made separately). Ship it!

Reply:
Thank you @tonycuadra ! I agree with the `timeDelta`, however the app type needs to consider the `manifest.xml` file configuration as well. I am afraid if we have `type` defaulting to `landscape` the user might forget to verify that needs to properly configure `manifest.xml` file. That might not be a problem since `main.js` is part of the template. :thinking:

Reply:
Good point (I was on the fence about type being optional anyway)